### PR TITLE
[Feature] weewx interceptor

### PR DESCRIPTION
**Pull request by MarcHagen**
# 

## 🗣 Description

Added [WeeWX-Interceptor](https://github.com/matthewwall/weewx-interceptor), this module allows you to intercept/receive data from a network weather station.

I have NOT added the sniff part (so the use of pypcap and such) as this is kinda complicated to do with Docker as it also requires a DNS/DHCP server.
Personally i find this out of the scope for the docker image, hench why i didn't add that dependency.

## 💭 Motivation and Context

It allows me to get data from my weather station and publish it over MQTT.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ This docker container can be used to quickly get a
 This container has the following WeeWX extensions installed:
 
 * [mqtt](https://github.com/weewx/weewx/wiki/mqtt)
+* [interceptor](https://github.com/matthewwall/weewx-interceptor)
```
Comment (felddy) [inline README.md]: Just need to sort this list: 

```suggestion
* [interceptor](https://github.com/matthewwall/weewx-interceptor)
* [mqtt](https://github.com/weewx/weewx/wiki/mqtt)
```

**Review by felddy — CHANGES_REQUESTED** [review]
Looks great!  Strong work!  💪 
Please see my one request for a change.

File: README.md (hunk 1 above)
Comment (MarcHagen) [reply README.md]: Updated branch with requested change.

**Review by felddy — APPROVED** [review]
Nice!  Thank you for the contribution!

**Comment by felddy** [comment]
@MarcHagen ... just a heads up... the release workflow is having issues building the image and publishing it.  This isn't caused by any changes in this PR, but will delay `4.1.1+a` from get published.   I'm trying to debug it now.

**Comment by felddy** [comment]
I was able to determine the root cause of the build errors.
See: #25

I've worked around this issue until it can be fixed upstream.  CI is now building the new `4.1.1` release and it should be pushed momentarily.    

Thank you again for your contribution!  💪

**Pull request merged**